src/app/modules/main/chat_section/controller.nim

```diff
@@ -700,3 +700,5 @@ proc getContractAddressesForToken*(self: Controller, symbol: string): Table[int,
 proc getCommunityTokenList*(self: Controller): seq[CommunityTokenDto] =
   return self.communityTokensService.getCommunityTokens(self.getMySectionId())
 
+proc getCheckCHannelpermissionResponses*(self: Controller, communityId: string, chatId: string) =
```

Comment:
```suggestion
proc getCheckChannelPermissionResponses*(self: Controller, communityId: string, chatId: string) =
```

Reply:
Wow, and nim isn't even complaining

Reply:
Good catch, thank you!

Reply:
Yeah I was almost sure this wouldn't even build... Almost...

How on earth is this possible? 😠